mlocal/frags/Makefile.stub

```diff
@@ -249,6 +249,11 @@ test:
 .PHONY: testall
 testall: check test
 
+.PHONY: rpm
+rpm:
+	@echo "Build RPM"
+	$(V)cd .. && dist/rpm/buildrpm
```

Comment:
What do you think of changing this to:

```
$(V)cd $(SOURCEDIR) && dist/rpm/buildrpm
```

Personally, I prefer setting the exact path there in the `cd`

Reply:
Ok, I made that change.

Reply:
Yeah I *had* to be done. Remember that mconfig -b /tmp/builddir could be passed.